### PR TITLE
Create README.md for Logistical Regression algo

### PR DESCRIPTION
**Pull request by genome21**
**Changes:**
Created a README.md for explanation and possible usages of the Logistical Regression algorithm.

### CODE REVIEW

File: code/artificial_intelligence/Logistic_Regression/README.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
+# Logistical Regression
+
+The **logistical regression** algorithm build upon linear regression and is basically a supervised classification algorithm. In a classification problem, the target variable (or output), y, can take only discrete values for given set of features (or inputs), X.
```
Comment (arnavb) [inline code/artificial_intelligence/Logistic_Regression/README.md]: Either capitalize both x and y, or neither. Don't mix cases.

File: code/artificial_intelligence/Logistic_Regression/README.md [hunk 2]
```diff
@@ -0,0 +1,21 @@
...
+We can also say that the target variable is categorical. Based on the number of categories, Logistic regression can be classified as:
+
+1. binomial: target variable can have only 2 possible types: “0” or “1” which may represent “win” vs “loss”, “pass” vs “fail”, “dead” vs “alive”, etc.
```
Comment (arnavb) [inline code/artificial_intelligence/Logistic_Regression/README.md]: These quotes seem unusual and special characters. Use normal quotes.

File: code/artificial_intelligence/Logistic_Regression/README.md [hunk 3]
```diff
@@ -0,0 +1,21 @@
...
+
+- Does NOT assume a linear relationship between the dependent variable and the independent variables, but it does assume linear relationship between the logit of the explanatory variables and the response.
+- Independent variables can be even the power terms or some other nonlinear transformations of the original independent variables.
+- The dependent variable does NOT need to be normally distributed, but it typically assumes a distribution from an exponential family (e.g. binomial, Poisson, multinomial, normal,…); binary logistic regression assume binomial distribution of the response.
+- The homogeneity of variance does NOT need to be satisfied.
+- Errors need to be independent but NOT normally distributed.
+- It uses maximum likelihood estimation (MLE) rather than ordinary least squares (OLS) to estimate the parameters, and thus relies on large-sample approximations.
+
+
+This article is obtained as reference from [Geeksforgeeks](http://www.geeksforgeeks.org/understanding-logistic-regression/)
```
Comment (arnavb) [inline code/artificial_intelligence/Logistic_Regression/README.md]: as _a_ reference ...

**Comment by AdiChat** [comment]
Looks great 😍 

Hope you are enjoying your journey 🚆  with **OpenGenus**

**Pull request merged**